ddl/partition.go

```diff
@@ -223,3 +223,14 @@ func checkAddPartitionTooManyPartitions(piDefs int) error {
 	}
 	return nil
 }
+
+func getPartitionIDs(table *model.TableInfo) []int64 {
```

Comment:
@tiancaiamao merged a PR with an exported method which allows you can get pid form that method.  By any chance, you could use that method.